gui/backend/search/search-api.coffee

```diff
@@ -243,6 +244,131 @@ exports.configureApp = (app, args) ->
+                allowNull: false
+
+            reviews_count:
+                type: Sequelize.BIGINT
+                allowNull: false
+
+            organization_score:
+                type: Sequelize.DOUBLE
+                allowNull: false
+
+            control_score:
+                type: Sequelize.DOUBLE
+                allowNull: false
+
+            underage_score:
+                type: Sequelize.DOUBLE
+                allowNull: false
+
+            movement_score:
+                type: Sequelize.DOUBLE
+                allowNull: false
+
+            overall_score:
+                type: Sequelize.DOUBLE
+                allowNull: false
+
+            state:
+                type: Sequelize.TEXT
+
+            city:
+                type: Sequelize.TEXT
+        }, {
+            indexes: [
+                {
+                    unique: true
+                    fields: ['phone_number']
+                },
+                {
+                    fields: ['overall_score']
+                },
+                {
+                    fields: ['movement_score']
+                },
+                {
+                    fields: ['underage_score']
+                },
+                {
+                    fields: ['control_score']
+                },
+                {
+                    fields: ['organization_score']
+                }
+            ]
+        })
+        Scores.sync()
+
+        app.get '/api/scores', (req, res, next) ->
+            Scores.findAll
+                order: req.query.sort_order 
+                limit: 200
+            .then (matches) ->
+                    #results = _.map matches, (item) ->
+                    #    phone_number: item.phone_number
+                    #    overall_score: item.overall_score                    
+                    results = matches
+                    res.send JSON.stringify(results)
+
+        Feedback = sequelize.define('feedback', {
+            doc_id:
+                type: Sequelize.TEXT
+                allowNull: false
+            mention_id:
+                type: Sequelize.TEXT
+            user_name:
+                type: Sequelize.TEXT
+            value:
+                type: Sequelize.TEXT
+        }, {
+            indexes: [
+                {
+                    unique: true
+                    fields: ['doc_id', 'mention_id']
+                }
+            ]
+        })
+
+        Feedback.sync()
+
+        app.get '/api/feedback/:doc_id', (req, res, next) ->
+            Feedback.findAll
+                where:
+                    doc_id: req.params.doc_id
+            .then (matches) ->
+                res.send JSON.stringify(matches)
+
+        app.post '/api/feedback', (req, res, next) ->
+            #if not req.user or not req.user.id
+            #    res
+            #        .status 400
+            #        .send 'You must log in to use the feedback service.'
+            #else
+                user_name = ''
```

Comment:
I'd forbid anonymous feedback to avoid data loss. why not just turn on google auth on 9001?